### ISSUE TITLE
Deleting multiple vertices

### ISSUE THREAD

**Issue by sinfex** [orientechnologies/orientdb#3177]
Any chance an enhancement for DELETE VERTEX syntax is possible to avoid using subquery targeting multiple RIDs? Considering how SELECT and UPDATE allows a collection of RIDs, one might expect the following to work:

```DELETE VERTEX [#12:34, #12:35]```

or, at least,

```DELETE VERTEX FROM [#12:34, #12:35]```

Instead, this seems to be the only way:

```DELETE VERTEX FROM (SELECT FROM [#12:34, #12:35])```

**Comment by luigidellaquila** [comment]
fixed in 2.1 snapshot

**Comment by sinfex** [comment]
How has this been fixed?

```
orientdb {db=test}> DELETE VERTEX [#13:2, #13:3]
```

> Error: com.orientechnologies.orient.core.sql.OCommandSQLParsingException: Error on parsing command at position #0: Class '[#13:2, #13:3]' was not found

```
orientdb {db=test}> DELETE VERTEX FROM [#13:2, #13:3]
```

> Error: com.orientechnologies.orient.core.command.OCommandExecutorNotFoundException: Cannot find a command executor for the command request: sql.

```
orientdb {db=test}> DELETE VERTEX FROM (SELECT FROM [#13:2, #13:3])
```

> Delete record(s) '2' in 0.015000 sec(s).
